docs/csharp/language-reference/keywords/extern-alias.md

```diff
@@ -31,6 +31,28 @@ You might have to reference two versions of assemblies that have the same fully-
   
  In the previous example, `GridV1::Grid` would be the grid control from `grid.dll`, and `GridV2::Grid` would be the grid control from `grid20.dll`.  
   
+## Using Visual Studio
+
+If you are using Visual Studio, aliase can be provided in similar way.
+
+Add reference of *grid.dll* and *grid20.dll* to your project in Visual Studio. Open a property tab and change the Aliases from global to GridV1 and GridV2 respectively.
```

Comment:
This should be a bit more instructive on how to add a reference. (If you check other articles, you'll see that we explain how to right-click on the project node, select add reference and so on.

Reply:
Yeah, I understand. initially adding this info looked out of main subject and redundant. But it could be like adding reference of doc page would worth.